### DISCUSSION TITLE
If backup fails, notitification fails with errors

### DISCUSSION THREAD

**Discussion by deanzod** [spatie/laravel-backup#1510] [Q&A]
I have the following in my kernel file

```php

        if (app()->environment('production')) {
            $schedule->command('backup:clean')->daily()->at('01:00');
            $schedule->command('backup:run --only-db')->daily()->at('01:20');
            $schedule->command('backup:run')->weekly()->at('01:30');
        }
```

This ran this morning and I received the clean confirmation email but the 2 backups never happened and no notification email arrived. I checked the log file and got this error....

```
production.ERROR: Undefined property: Spatie\Backup\Events\BackupHasFailed::$exception {"exception":"[object] (ErrorException(code: 0): Undefined property: Spatie\\Backup\\Events\\BackupHasFailed::$exception at /var/www/html/vendor/spatie/laravel-backup/src/Notifications/Notifications/BackupHasFailedNotification.php:24)
```

If I connect to the server with tinkerwell and run `Artisan::call('backup:run --only-db');` the backup works fine and confirmation email arrives.

Firstly, any idea what is wrong...and secondly, is there a way to get more informative error messages if this isn't a package bug?

**Comment by PaolaRuby** [comment]
Maybe folder permissions
